src/test/java/core/basesyntax/dataprocessor/DefaultDataOperationStrategyTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.dataprocessor;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class DefaultDataOperationStrategyTest {
+
+    private DefaultDataOperationStrategy strategy;
+
+    @BeforeEach
+    void setUp() {
+        OperationHandler balanceHandler = (fruit, quantity) -> {};
+        OperationHandler supplyHandler = (fruit, quantity) -> {};
+        OperationHandler purchaseHandler = (fruit, quantity) -> {};
+        OperationHandler returnHandler = (fruit, quantity) -> {};
+
+        strategy = new DefaultDataOperationStrategy(
+                Map.of(
+                        Operation.BALANCE, balanceHandler,
+                        Operation.SUPPLY, supplyHandler,
+                        Operation.PURCHASE, purchaseHandler,
+                        Operation.RETURN, returnHandler
+                )
+        );
```

Comment:
What is the purpose of it?)
```suggestion
        strategy = new DefaultDataOperationStrategy(
                Map.of(
                        Operation.BALANCE, new BalanceHandler(),
                        Operation.SUPPLY, new SupplyHandler(),
                        Operation.PURCHASE, new PurchaseHandler(),
                        Operation.RETURN, new ReturnHandler()
                )
        );
```